satpy/readers/fci_l1c_nc.py

```diff
@@ -652,3 +647,10 @@ def calibrate_rad_to_refl(self, radiance, key):
 
         res = 100 * radiance * np.pi * sun_earth_distance ** 2 / cesi
         return res
+
+
+def _ensure_dataarray(arr):
+    if not isinstance(arr, xr.DataArray):
+        attrs = dict(arr.attrs.items()).copy()
```

Comment:
Is the `.items()` and `dict` needed to convert this to a regular dictionary? Does doing `dict(arr.attrs).copy()` not work? Or `arr.attrs.copy()`?

Reply:
Hmm. I wonder why I've had the `.items()` in there. Plain `dict(arr.attrs).copy()` works, but with `h5netcdf`
```python
In [5]: fid.attrs.copy()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[5], line 1
----> 1 fid.attrs.copy()

AttributeError: 'Attributes' object has no attribute 'copy'
```
I'll remove the `.items()`.

Reply:
Removed in 9e5186c0e